.release-please-manifest.json

```diff
@@ -1,4 +1,5 @@
 {
   "stack": "2.10.2",
-  "argus-config": "1.4.1"
+  "argus-config": "1.4.1",
+  "secret-store": "0.1.0"
```

Comment:
You need a new package added to https://github.com/chanzuckerberg/argo-helm-charts/blob/main/release-please-config.json#L22 for this to work

Reply:
Done!